XenCert/StorageHandler.py

```diff
@@ -2781,103 +2800,274 @@ def MetaDataTests(self):
         Print("MetaDataTests not applicable to NFS SR type.")
         return False
 
+    def PoolTests(self):
```

Comment:
Why are we adding pool tests for NFS SRs? They mainly test multipathing which doesn't make sense for file based SRs.